src/cli/commands/install.js

```diff
@@ -1223,11 +1223,6 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
   }
 
   await install(config, reporter, flags, lockfile);
-
-  // Let time for all the output to be printed and then hard exit.
-  // This fixes an issue where a hanging request would keep node alive.
-  // eslint-disable-next-line no-process-exit
-  setTimeout(() => process.exit(process.exitCode || 0), 1000);
```

Comment:
I suppose that removing this line will re-introduce the issue it fixed. Have you considered lowering this timeout to something like 100ms?

Reply:
Moved it behind a flag.